geoips/plugins/modules/readers/ahi_hsd.py

```diff
@@ -157,10 +156,10 @@
 name = "ahi_hsd"
 
 
-class AutoGenError(Exception):
-    """Raise exception on geolocation autogeneration error."""
+# class AutoGenError(Exception):
```

Comment:
Couldn't this be used from another module somewhere ?

Reply:
Leaving this commented out for now. This was overwritten as it was already imported into the file. I'll let you decide what to do on this one!